cfme/common/provider.py

```diff
@@ -437,15 +433,7 @@ def _on_detail_page(self):
 
     def load_details(self, refresh=False):
```

Comment:
What are you doing with refresh here? We hsould still honour the ask to refresh until we remove it.